### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by alinnegrea**
Homework-2 submission on GitHub

### CODE REVIEW

File: java-grupa8/homework-2/src/main/java/ro/siit/java8/SalesRepresentatives.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package ro.siit.java8;
+
+/**
+ * @author Alin_Negrea
+ * Defines the Sales Representatives class characteristics:
+ * name,
+ * number of sales
+ * and quota for sale.
+ */
+public class SalesRepresentatives {
```
Comment (andrei-hegedus) [inline java-grupa8/homework-2/src/main/java/ro/siit/java8/SalesRepresentatives.java]: Clasele sunt scrise la singular: SalesRepresentative.

File: java-grupa8/homework-2/src/main/java/ro/siit/java8/main.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
+package ro.siit.java8;
+
+/**
+ * @author Alin_Negrea
+ * A bubble  sort algorithm was implemented to sort an array of Sales Representative objects in descending order.
+ * The sorting was done based on the amount of revenue generated via sales.
+ * Each representative has a number of sale, and a quota/sale.
+ *
+ */
+public class Main {
```
Comment (andrei-hegedus) [inline java-grupa8/homework-2/src/main/java/ro/siit/java8/main.java:10]: Java class files must have the same name as the class: Main.java and not main.java.

File: java-grupa8/homework-2/src/main/java/ro/siit/java8/main.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        member[j] = memberTemp;
+    }
+
+    /**
+     * Bubble sort method is implemented. The return is an descending order of the Sales Representatives based on revenue.
+     * @param member
+     * @return
+     */
+    public static SalesRepresentatives[] sortSalesRepresentatives(SalesRepresentatives[] member) {
+        for (int i = 0; i < 10; i++) {
```
Comment (andrei-hegedus) [inline java-grupa8/homework-2/src/main/java/ro/siit/java8/main.java]: This is not Bubble Sort. You implemented a brute force matching of all numbers with all numbers.
Bubble sort has extra conditions, please look over it again: https://en.wikipedia.org/wiki/Bubble_sort

**Review by andrei-hegedus — CHANGES_REQUESTED** [review]
See my comments.

File: java-grupa8/homework-2/src/main/java/ro/siit/java8/SalesRepresentatives.java (hunk 1 above)
Comment (alinnegrea) [reply java-grupa8/homework-2/src/main/java/ro/siit/java8/SalesRepresentatives.java]: Done

File: java-grupa8/homework-2/src/main/java/ro/siit/java8/main.java (hunk 2 above)
Comment (alinnegrea) [reply java-grupa8/homework-2/src/main/java/ro/siit/java8/main.java:10]: Done

File: java-grupa8/homework-2/src/main/java/ro/siit/java8/main.java (hunk 3 above)
Comment (alinnegrea) [reply java-grupa8/homework-2/src/main/java/ro/siit/java8/main.java]: I tried to use a for for the Bubble Sort. You are right the method was not a Bubble sort.. i read the extra conditions and modified the code.

**Pull request merged**